_includes/manuals/1.12/4.1.2_roles_and_permissions.md

```diff
@@ -2,23 +2,15 @@ A user's access to the features of Foreman are constrained by the **permissions*
 
 Note: a user with global admin enabled is not restricted by the authorization system. This is the default for installations that do not have :login:true in _config/settings.yml_.
 
-A logged in user will be granted the **Anonymous** role plus one or more additional roles. The permissions and filters associated with these roles are aggregated and determine the final permission set.
+A logged in user will be granted the **Default Role** role plus one or more additional roles. The permissions and filters associated with these roles are aggregated and determine the final permission set.
 
 Roles may be administered by users with admin privileges or regular users with 'edit_roles' permission. In order to add new filters and permissions to a role, regular users must have the 'create_filters' permission.
 
 #### Roles
 
 These may be created, deleted and edited on the **Roles** page. Each role will contain permission filters, which define the actions allowed in a certain resource. Once your role is created, you can associate it with one or more users and user groups.
 
-There are two builtin system roles
-
-
-1. Anonymous: This is a set of permissions that every user at your installation will be granted, irrespective of any other roles that they have.
-
-
-2. Default user: When a new Role is created this set of permissions are used as the template for the Role. The name is somewhat misleading but basically an ordinary default user who was assigned this Role would have these permissions set.
-
-
+There is one built-in system role, 'Default Role'. This is a set of permissions that every user at your installation will be granted, irrespective of any other roles that they have.
```

Comment:
1) ditto re case.
2) "irrespective of" => "in addition to"
3) I'd perhaps leave out "at your installation".